CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 See below for Changelog examples.
 
+### 0.6.5
```

Comment:
We don't need to add a version number heading in this PR, that happens when we do a release